apps/pkt-gen/pkt-gen.c

```diff
@@ -2401,13 +2543,27 @@ start_threads(struct glob_arg *g) {
 		} else {
 			t->affinity = -1;
 		}
+		/* only thread 0 can send ARP */
+		t->garp = (g->garp && t->me == 0 && g->af == AF_INET);
 		/* default, init packets */
 		initialize_packet(t);
 	}
 	/* Wait for PHY reset. */
-	D("Wait %d secs for phy reset", g->wait_link);
-	sleep(g->wait_link);
-	D("Ready...");
+	if (g->wait_link || g->dev_type != DEV_NETMAP) {
+		D("Wait %d secs for phy reset", g->wait_link);
+		sleep(g->wait_link);
+		D("Ready...");
+	} else {
+		D("Wait for phy reset");
+		for (i = 5*4; i > 0; i--) {
```

Comment:
20?

Reply:
1000/250*5

Reply:
You are waiting for 5 seconds here (max)
````
D("Wait up to 5 seconds for phy reset")
````